static/js/map.js

```diff
@@ -840,6 +842,11 @@ function customizePokemonMarker(marker, item, skipNotification) {
                 marker.setAnimation(google.maps.Animation.BOUNCE)
             }
         }
+        if (Store.get('remember_show_ivzero') && perfection === 0) {
+            if (marker.animationDisabled !== true) {
```

Comment:
This PR will only make the markers bounce, it will not send a notification. Is this intended or a mistake?